packages/ibm-products/src/components/AddSelect/AddSelectBody.tsx

```diff
@@ -275,11 +351,11 @@ export let AddSelectBody = forwardRef(
             </div>
             {showSort && (
               <AddSelectSort
-                items={itemsToDisplay}
+                // items={itemsToDisplay}
                 setSortAttribute={setSortAttribute}
                 setSortDirection={setSortDirection}
-                sortAttribute={sortAttribute}
-                sortDirection={sortDirection}
+                // sortAttribute={sortAttribute}
+                // sortDirection={sortDirection}
```

Comment:
just wondering, are these supposed to stay commented out?

Reply:
same comment